llvm/utils/release/test-release.sh

```diff
@@ -417,6 +417,10 @@ function configure_llvmCore() {
     # building itself and any selected runtimes in the second phase.
     if [ "$Phase" -lt "2" ]; then
       runtime_list=""
+      # compiler-rt built-ins needed on AIX to have a functional Phase 1 clang.
```

Comment:
nit: the library name is `builtins`
```suggestion
      # compiler-rt builtins is needed on AIX to have a functional Phase 1 clang.
```